pkg/cmd/cli/describe/describer.go

```diff
@@ -377,7 +377,11 @@ func (d *BuildConfigDescriber) Describe(namespace, name string) (string, error)
 	if err != nil {
 		return "", err
 	}
-	buildList, err := d.Builds(namespace).List(labels.SelectorFromSet(labels.Set{buildapi.DeprecatedBuildConfigLabel: name}), fields.Everything())
+	buildConfigLables := labels.Set{
+		buildapi.BuildConfigLabel: name,
```

Comment:
You can't use selector from set, since that ANDs the labels together. You need an OR, which I don't think is supported